components/ProjectQuality.vue

```diff
@@ -0,0 +1,53 @@
+<template>
+  <div>
+    <div v-for="tag in statusTags" :key="tag" class="flex justify-end pb-2 px-4">
+      <div class="px-4">
+        {{ tag.desc }}
+        <span :class="`px-3 py-1 rounded-full text-sm font-semibold text-white ${tag.color} shadow-lg`">
+          {{ tag.label }}
+        </span>
+      </div>
+    </div>
+    <div class="flex justify-end pb-2 px-4">
+      <span>Maturity: </span
+      ><span
```

Comment:
What formatting is this?
```suggestion
      <span>Maturity: </span>
      <span
```

Reply:
that's the linter unfortunately...